recipe/meta.yaml

```diff
@@ -26,9 +27,9 @@ requirements:
   run:
     - python >=3.6
     - cryptography >=3.2
-    - google-auth
+    - google-auth >=1.0.0
     - requests-oauthlib >=0.7.0
-    - click
+    - click >=6.0.0
 
 test:
   requires:
```

Comment:
I think we don't need `python >=3.6` in test/requires. It will be there implicitly from `run`